src/core/BufferAttribute.js

```diff
@@ -319,6 +312,22 @@ Object.assign( BufferAttribute.prototype, {
 
 		return new this.constructor( this.array, this.itemSize ).copy( this );
 
+	},
+
+	dispose: function () {
```

Comment:
Can you please also update the documentation page of `BufferAttribute`? You have to add the `dispose()` method similar to the other pages (e.g. https://threejs.org/docs/#api/en/core/BufferGeometry.dispose).

Reply:
i have updated the doc and added InterleavedBufferAttribute.dispose method like BufferAttriubte